frontend/common/src/helpers/format.tsx

```diff
@@ -20,7 +20,9 @@ export const strong = (text: React.ReactNode): React.ReactNode => (
  * @param text text to wrap.
  */
 export const hidden = (text: React.ReactNode): React.ReactNode => (
-  <span data-h2-visibility="base(invisible)">{text}</span>
+  <span data-h2-visibility="base(invisible)" style={{ whiteSpace: "pre" }}>
```

Comment:
Replace style attribute with hydrogen attribute
```suggestion
  <span data-h2-visibility="base(invisible)" data-h2-white-space="base(pre)">
```

Accordion to @substrae, this should work. Does it?

Reply:
no it doesn't
![image](https://user-images.githubusercontent.com/40485260/203164761-05d016c6-a54b-4c25-94f9-797002e9facb.png)